app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -370,7 +370,7 @@ class HistoryFragment : LibraryPageFragment<History>(), UserInteractionHandler {
     @Suppress("UnusedPrivateMember")
     private suspend fun syncHistory() {
         val accountManager = requireComponents.backgroundServices.accountManager
-        accountManager.syncNow(SyncReason.User)
+        accountManager.syncNow(reason = SyncReason.User, debounce = true)
```

Comment:
Same comment here as in bookmarks.

Reply:
Sure, I'll update to use the history engine.